test/ETO/ETOCommitment.js

```diff
@@ -416,18 +428,36 @@ contract("ETOCommitment", ([, admin, company, nominee, ...investors]) => {
 
     it("with changing ether price during ETO", async () => {
       await skipTimeTo(publicStartDate.add(1));
+      // set some round ETH price to minimize rounding errors in the contract
+      await gasExchange.setExchangeRate(etherToken.address, euroToken.address, Q18, {
+        from: admin,
+      });
       const tokenprice = tokenTermsDict.TOKEN_PRICE_EUR_ULPS;
-      const ticket = Q18.mul(107.61721).add(1);
+      const ticket = Q18.mul("1070.61721").add(1);
       await investAmount(investors[1], ticket, "ETH", tokenprice);
-      const newPrice = Q18.mul(defEthPrice.mul(2));
+      // set double price
+      const newPrice = Q18.mul(2);
       await gasExchange.setExchangeRate(etherToken.address, euroToken.address, newPrice, {
         from: admin,
       });
       await investAmount(investors[2], ticket, "ETH", tokenprice);
       // token price increased twice
       const i1ticket = await etoCommitment.investorTicket(investors[1]);
       const i2ticket = await etoCommitment.investorTicket(investors[2]);
-      expect(i1ticket[2].mul(2)).to.be.bignumber.eq(i2ticket[2]);
+      // but it does not mean that previous ticket amount is doubled due to rounding once, not twice
+      // i1ticket[2].mul(2) != i2ticket[2]
+      expect(
```

Comment:
as a whole this test is not really easy to understand.

>       // token price increased twice
we mean it increased by factor of two, correct?￼

[2] refers to equityTokenAmount, so maybe a helpful comment could be: 
```// verify that same amount at double price buys almost same amount of equityToken (slightly less because of rounding)```

Reply:
you are right here! btw this test was insane to track all rounding errors. there's internal price conversion done with the inverse of the rate in the contract. setting price of ETH to 2 EUR removes rounding problems at that stage and I could write stricter test